### DISCUSSION TITLE
cn/2022/09/selfish-and-greedy/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#38] [Comments]
# cn/2022/09/selfish-and-greedy/

适当的自私，但不应贪婪。 学了这么多年管理，除了印象最深且还能在工作中不时提起的马斯洛需求层次理论之外，还有就是亚当斯密的经济人假设了。经济人

https://leovan.me/cn/2022/09/selfish-and-greedy/

**Comment by March-h-are** [comment]
寬以待人，嚴以律己，己所不欲，勿施於人，
人非聖賢，孰能無過，過而能改，善莫大焉。
